### PR TITLE
:seedling: Add step to validate the dist folder content

### PR DESCRIPTION
**Pull request by UlisesGascon**
### Main Changes

This change adds a new step in the ci pipeline. The idea is to ensure that the `dist/` folder is up to date with the source code of the project. In the past quite often I forgot to do this step as it is a manual process that is not validated on PR submission.

In this PR we include in the dist folder changes in the dependencies and critical changes such as https://github.com/ossf/scorecard-monitor/pull/78

#### Demo

When the folder is not updated:

Screenshot from 2024-06-21 08-20-57

When the folder is updated:

Screenshot from 2024-06-21 08-22-28

### CODE REVIEW

File: dist/issue.ejs [hunk 1]
```diff
@@ -16,4 +16,4 @@ There are changes in the following repositories:
 | [<%= score.org %>/<%= score.repo %>](https://<%= score.platform %>/<%= score.org %>/<%= score.repo %>) | [<%= score.commit.slice(0, 7) %>](https://<%= score.platform %>/<%= score.org %>/<%= score.repo %>/commit/<%= score.commit %>) | <% if (!renderBadge) { -%><%= score.score %> <%_ } -%> <%_ if (renderBadge) { -%> [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/<%= score.org %>/<%= score.repo %>/badge)](https://api.securityscorecards.dev/projects/github.com/<%= score.org %>/<%= score.repo %>) <%_ } -%> | <%= score.currentDiff || 0 %><%_ if (score.prevCommit) { -%> / [Details](<%= getReportUrl(score.org, score.repo, score.commit, score.prevCommit) %>)<%_ } -%> | [View](<%= getReportUrl(score.org, score.repo, score.commit) %>) | [Fix it](https://app.stepsecurity.io/securerepo?repo=<%= score.org %>/<%= score.repo %>) |
 <%_ }); -%> 
 
-_Report generated by [UlisesGascon/openssf-scorecard-monitor](https://github.com/UlisesGascon/openssf-scorecard-monitor)._
\ No newline at end of file
+_Report generated by [ossf/scorecard-monitor.com/ossf/scorecard-mossf/scorecard-monitor
```
Comment (lelia) [inline dist/issue.ejs:19]: I think we can leave these changes out of the PR due to https://github.com/ossf/scorecard-monitor/pull/83

File: dist/report.ejs [hunk 2]
```diff
@@ -12,4 +12,4 @@
 | [<%= score.org %>/<%= score.repo %>](https://<%= score.platform %>/<%= score.org %>/<%= score.repo %>) | [<%= score.commit.slice(0, 7) %>](https://<%= score.platform %>/<%= score.org %>/<%= score.repo %>/commit/<%= score.commit %>) | <% if (!renderBadge) { -%><%= score.score %> <%_ } -%><% if (renderBadge) { -%> [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/<%= score.org %>/<%= score.repo %>/badge)](https://api.securityscorecards.dev/projects/github.com/<%= score.org %>/<%= score.repo %>) <%_ } -%> | <%= score.date %> | <%= score.currentDiff || 0 %> <%_ if (score.prevCommit) { -%> / [Details](<%= getReportUrl(score.org, score.repo, score.commit, score.prevCommit) %>)<%_ } -%> | [View](<%= getReportUrl(score.org, score.repo, score.commit) %>) | [Fix it](https://app.stepsecurity.io/securerepo?repo=<%= score.org %>/<%= score.repo %>) |
 <%_ }); -%>
 
-_Report generated by [UlisesGascon/openssf-scorecard-monitor](https://github.com/UlisesGascon/openssf-scorecard-monitor)._
\ No newline at end of file
+_Report generated by [ossf/scorecard-monitor](https://github.com/ossf/scorecard-monitor)._
```
Comment (lelia) [inline dist/report.ejs:15]: Same comment as [above](https://github.com/ossf/scorecard-monitor/pull/84/files#r1649427262)

File: .github/workflows/code_health.yml [hunk 3]
```diff
@@ -28,4 +28,16 @@ jobs:
...
+      run: |
+        FILES=$(git status -s dist/)
+        if [ -n "$FILES" ]; then
+          echo "You need to include the changes in the dist folder, execute: npm run build and commit the changes in the dist folder"
+          echo "Uncommitted files:"
+          echo "$FILES"
+          echo "Diff content:"
+          git diff dist/
+          exit 1
+        fi
```
Comment (lelia) [inline .github/workflows/code_health.yml:43]: [NIT] Missing newline

File: dist/issue.ejs (hunk 1 above)
Comment (UlisesGascon) [reply dist/issue.ejs:19]: Yes, #83 will cover it :)

File: .github/workflows/code_health.yml (hunk 3 above)
Comment (UlisesGascon) [reply .github/workflows/code_health.yml]: Solved in https://github.com/ossf/scorecard-monitor/pull/84/commits/d21a005ee302e472b039cfc367c4255a92ff91b2

**Review by lelia — APPROVED** [review]
LGTM

**Review by justaugustus — APPROVED** [review]

**Pull request merged**